service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -118,5 +118,53 @@
     "usageRules":[
       
     ]
+  },
+  {
+    "quipGroup":"HIGH_FIVE",
+    "text":"You're pretty awesome! I'd high five you, if I had hands!",
+    "ssml":"You're pretty awesome!<break /> I'd high five you, if I had hands!",
+    "usageRules":[
+      
+    ]
+  },
+  {
+    "quipGroup":"SO_HOT",
+    "text":"You're so hot that the sun is jealous.",
+    "ssml":"You're so hot that the sun is jealous.",
+    "usageRules":[
+      
+    ]
+  },
+  {
+    "quipGroup":"SO_HOT",
+    "text":"The earth has been heating up since your birth.  Scientists conclude that you're so hot you cause global warming.",
```

Comment:
"The earth has been heating up since your birth.  Scientists HAVE CONCLUDED that you're so hot you cause global warming."

Does it sound better with this conjugation?